src/main/java/com/ksm/robolo/roboloapp/rest/RegistrationController.java

```diff
@@ -0,0 +1,33 @@
+package com.ksm.robolo.roboloapp.rest;
+
+import com.ksm.robolo.roboloapp.domain.UserEntity;
+import com.ksm.robolo.roboloapp.services.UserService;
+import com.ksm.robolo.roboloapp.tos.UserTO;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class RegistrationController {
+
+    private final UserService userService;
+
+    @Autowired
+    public RegistrationController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @PostMapping(value = "/register")
+    public String registerUser(@ModelAttribute UserTO userTO) {
+        UserEntity userEntity = new UserEntity();
+        userEntity.setName(userTO.getName());
+        userEntity.setSurname(userTO.getSurname());
+        userEntity.setUsername(userTO.getUsername());
+        userEntity.setPassword(userTO.getPassword());
+        userEntity.setEmail(userTO.getEmail());
+        userService.saveUser(userEntity);
+        return "registered";
```

Comment:
Co się tu stanie taki użytkownik już istnieje i poleci wyjątek z JDBC?

Reply:
na innym branchu to ogarniam, tutaj tylko tak dummy teraz zostało zrobione